dascore/utils/chunk_plan.py

```
@@ -1258,6 +1260,225 @@ def build_chunk_plan(
+                int(i): [x[1 : -len("_def_key")] for x in aux_keys if row[x]]
+                for i, row in varied.iterrows()
+                if row.any()
+            }
+    if "_dtype" in sorted_df.columns:
+        all_dtypes = sorted_df["_dtype"]
+        dtypes = by_output["_dtype"]
+        uniform = dtypes.nunique(dropna=False).to_numpy() == 1
+        first = dtypes.first().to_numpy(dtype=object)
+        combined = [
+            first[i] if uniform[i] else _combined_dtype(all_dtypes.iloc[a : a + n])
+            for i, (a, n) in enumerate(zip(seg_starts, sizes))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)